lib/rspec/retry/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module RSpec
   class Retry
-    VERSION = '0.6.2'
+    VERSION = '0.7.0'
```

Comment:
What behaviour has been added to warrant a minor bump?

Reply:
I considered the ruby upgrade to be a minor bump instead of any bugfix. Wdyt?

Reply:
I guess it lets you use ruby 3 stuff.